src/pages/HomePage.spec.js

```diff
@@ -0,0 +1,9 @@
+import { render, screen } from '@testing-library/react'
+import HomePage from './HomePage'
+
+describe('HomePage', () => {
+  it('renders', () => {
+    render(<HomePage />)
+    expect(screen.getByText('INSIDE JOB')).toBeInTheDocument()
```

Comment:
You could also use getByRole('heading', { level: 2 }) (if the header is not a h2 as well)

Reply:
Its only a placeholder yet. there will be a logo a small information about the app later :)